spring-kafka/src/main/java/org/springframework/kafka/core/CompositeKafkaStreamsCustomizer.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017-2018 the original author or authors.
```

Comment:
This class has been created in this `2018`, so, no any past in the Copyright, please.
Just the actual truth.